### PR TITLE
Use Ctrl+Mouse wheel to increase / decrease the editor font size

### LINKED ISSUE #244: Feature Request: change the text size (zoom in/out)
It would be nice having the option to change the editor text size.
If you give me some hint, I can try to implement it.

### PR DESCRIPTION
**Pull request by xgouchet**
Fixes #244

### CODE REVIEW

File: ReText/editor.py [hunk 1]
```diff
@@ -401,6 +401,23 @@ def updateTextStatistics(self):
 			wordCount += 1
 		self.statistics = (wordCount, alphaNumCount, characterCount)
 
+	def wheelEvent(self, e: QWheelEvent):
```
Comment (mitya57) [inline ReText/editor.py]: This requires Python 3.5, but the README says that ReText works with 3.2 or higher. Please update the README or remove this.

File: ReText/editor.py [hunk 2]
```diff
@@ -401,6 +401,23 @@ def updateTextStatistics(self):
...
+		modifiers = QGuiApplication.keyboardModifiers()
+		if modifiers == Qt.ControlModifier:
+			font = globalSettings.editorFont
+			size = font.pointSize()
+			scroll = e.angleDelta().y()
+			if (scroll > 0):
+				size = size+1
+			elif (scroll < 0):
+				size = size-1
```
Comment (mitya57) [inline ReText/editor.py]: `size += 1` / `size -= 1` would be a bit cleaner.

Also no parentheses are needed around `scroll > 0` / `scroll < 0` in Python (unlike C-like languages).

File: ReText/editor.py (hunk 1 above)
Comment (xgouchet) [reply ReText/editor.py]: Sorry bout that, I'm used to typed languages :/

File: ReText/editor.py (hunk 2 above)
Comment (xgouchet) [reply ReText/editor.py]: Yep, habits from other languages :/

**Comment by mitya57** [comment]
Merci!

**Comment by xgouchet** [comment]
Anytime ;)

**Pull request merged**